Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- RE2.Native will now compile in all environments with the latest Windows SDK 10.0.* installed. [#607](https://github.com/microsoft/sarif-pattern-matcher/pull/607). Our current release pipelines build NuGet packages with Windows SDK version 10.0.22000.
```

Comment:
>10.0.22000 [](http://example.com/codeflow?start=241&length=10)

@eddynaka, we need to update this version w/what's actually used in our release/deployment pipeline build agents.

Reply:
The version that you mentioned is the version that we will be using.
So, the text is accurate.

Thanks for updating it.